app/models/user.rb

```diff
@@ -24,6 +24,7 @@ class User < ApplicationRecord
   has_many :comments, dependent: :nullify
   has_many :comment_threads_deleted, class_name: 'CommentThread', foreign_key: :deleted_by_id, dependent: :nullify
   has_many :comment_threads_locked, class_name: 'CommentThread', foreign_key: :locked_by_id, dependent: :nullify
+  has_many :filters, dependent: :destroy
```

Comment:
Tests fail because this doesnt reference the category_filter_defaults, needs:

```suggestion
  has_many :filters, dependent: :destroy
  has_many :category_filter_defaults, dependent: :destroy
```